CONTRIBUTING.md

```diff
@@ -41,6 +41,10 @@ npx nx affected:lint
 npx nx code-pushup -- collect
 ```
 
+### Testing
+
+To skip long-running tests on your local environment set the following environment variable `SKIP_LONG_TESTS=true`.
```

Comment:
I think the flag should work in the opposite way - as a contributor I should not be required to know about and set the flag if I want a smooth DX - in other words, one should opt in, not opt out.

Reply:
That way we would have to strictly configure it. I would rather prefer to have our CI run by default all tests instead of  a custom configuration that lives in the `action.yml` file.

Is there a way to detect if we are on a local setup?

Reply:
> Is there a way to detect if we are on a local setup?

The best heuristic for that is to check if `CI` environment variable is defined.

Reply:
I could use that.. but then i would mess around with that var locally. 

Should I:
- A) keep as is
- B) refactor to derive the skip decision form `CI` (which would mean i have to set it to true locally if I want to run all tests.. Naming is not really intuitive.)

Reply:
Let's first agree on what the desired outcome is :sweat_smile: 

1. Should long-running tests run in _CI_?
2. Should long-running tests run _locally by default_?

Current implementation assume the answer to both is _yes_, but @Tlacenka is suggesting something different (at least for local, not sure about CI).

As far as I'm concerned, I'm happy to go with the current implementation and tweak it later if we find it impractical. What do you guys think?

Regarding the implementation, we can create a helper function in `test-utils`, which could check multiple environment variables if needed.

Reply:
As a compromise I can implement the helper already and Go with `C`I || `SKIP_LONG_TESTS`, and we polish later.

Reply:
Updated the code with the helper function

Reply:
Isn't the code already skipping the long tests locally?